src/components/NewsList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const NewsContent = (props) => {
```

Comment:
Is using () => {} a paradigm we should follow throughout?

Reply:
For declaring stateless components, it is mostly a matter of style I think. 
With arrow functions,, you can omit `{}` and the `return` keyword which makes it more concise. A downside would be that they aren't hoisted, so any references to components must come after the definition. I think we should just make a decision and stick with it keep the code base consistent. 
Outside of components they actually have some other differences, specifically with how they bind `this`.

Reply:
Thanks for explaining this! I'm happy with using arrow functions throughout. I think the non-hoisted bit is not a bit deal. I can make a ticket for everything else to be converted.